src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -456,6 +456,36 @@ public function theVisitorShouldSeeAsThePriceOfTheProductInTheChannel(
         ));
     }
 
+    /**
+     * @Then /^I should not be able to select the "([^"]*)" ([^\s]+) option value$/
```

Comment:
I'm not sure but if `$optionValue` isn't nullable therefore we shouldn't use `"([^"]+)"` instead of `"([^"]*)"?